### ISSUE TITLE
How to convert data format

### ISSUE THREAD

**Issue by qjl1244281167** [MPC-Berkeley/dlp-dataset#8]
I want to use this data set to achieve target detection. I want to get the label box of each instance (such as vehicles, pedestrians). How do I convert it? I want the format of VOC or COCO. If there is a YOLO format, that would be more alright

**Comment by XuShenLZ** [comment]
Hi, I think you might want to have the raw video & annotation format of the data. Have you requested it? https://sites.google.com/berkeley.edu/dlp-dataset

**Comment by qjl1244281167** [comment]
> 您好，我想您可能想要数据的原始视频和注释格式。你要求了吗？https://sites.google.com/berkeley.edu/dlp-dataset
Yes, I have requested the dataset and I have received a download link for the dataset, but I don't quite understand the meaning of the content in the annotation files for these datasets and I don't know how to use it
